sql-statements/sql-statement-analyze-table.md

```diff
@@ -1,16 +1,18 @@
 ---
-title: ANALYZE TABLE | TiDB SQL Statement Reference
-summary: An overview of the usage of ANALYZE TABLE for the TiDB database.
+title: ANALYZE | TiDB SQL Statement Reference
+summary: An overview of the usage of ANALYZE for the TiDB database.
 category: reference
 aliases: ['/docs/dev/reference/sql/statements/analyze-table/']
 ---
 
-# ANALYZE TABLE
+# ANALYZE
 
-This statement updates the statistics that TiDB builds on tables and indexes. It is recommended to run `ANALYZE TABLE` after performing a large batch update or import of records, or when you notice that query execution plans are sub-optimal.
+This statement updates the statistics that TiDB builds on tables and indexes. It is recommended to run `ANALYZE` after performing a large batch update or import of records, or when you notice that query execution plans are sub-optimal.
 
 TiDB will also automatically update its statistics over time as it discovers that they are inconsistent with its own estimates.
 
+Currently, TiDB collects statistical information in two ways: full collection (implemented using `ANALYZE TABLE` statement) and incremental collection (implemented using the `ANALYZE INCREMENTAL TABLE` statement). For detailed usage of these two statements, refer to [introduction to statistics](/statistics.md)
```

Comment:
s/using/with|by

Reply:
`using` can be more direct than `with` or `by`. For example, `The full collection is implemented using ...`.